packages/zipkin-instrumentation-express/src/wrapExpressHttpProxy.js

```diff
@@ -18,7 +18,7 @@ class ExpressHttpProxyInstrumentation {
       const clientTraceId = this.tracer.createChildId();
       this.tracer.setId(clientTraceId);
 
-      const proxyReqWithZipkinHeaders = Request.addZipkinHeaders(proxyReq, clientTraceId);
+      const proxyReqWithZipkinHeaders = this.tracer.injector(proxyReq);
```

Comment:
I'd rather pass the clientTraceId explicitly instead of having the injector to know about the scope.